test/images.js

```diff
@@ -1,9 +1,11 @@
 const t = require('../test-lib/test.js');
-const assert = require('assert');
-const fs = require('fs');
+const assert = require('assert/strict');
+const fs = require('fs-extra');
```

Comment:
Do we need this dependency, modern fs has a lot of cool toys:

https://nodejs.org/api/fs.html#fsrmpath-options-callback

(Available at least back to node 18)